diff --git a/trac_ik_kinematics_plugin/include/trac_ik/trac_ik_kinematics_plugin.hpp b/trac_ik_kinematics_plugin/include/trac_ik/trac_ik_kinematics_plugin.hpp
index 8f0320f..3bcc217 100644
--- a/trac_ik_kinematics_plugin/include/trac_ik/trac_ik_kinematics_plugin.hpp
+++ b/trac_ik_kinematics_plugin/include/trac_ik/trac_ik_kinematics_plugin.hpp
@@ -31,8 +31,8 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef TRAC_IK_KINEMATICS_PLUGIN_
 #define TRAC_IK_KINEMATICS_PLUGIN_
 
-#include <moveit/kinematics_base/kinematics_base.hpp>
-#include <moveit/robot_model/robot_model.hpp>
+#include <moveit/kinematics_base/kinematics_base.h>
+#include <moveit/robot_model/robot_model.h>
 #include <kdl/chain.hpp>
 #include <kdl/jntarray.hpp>
 
